common/content-api.js

```diff
@@ -214,7 +214,7 @@ function getFundingProgrammes({
     ]).then((responses) => {
         const [enResults, cyResults] = responses.map(mapAttrs);
         return {
-            meta: head(responses).meta,
+            meta: locale === 'en' ? head(responses).meta : responses[1].meta,
             result: mergeWelshBy('slug')(locale, enResults, cyResults),
         };
```

Comment:
Wondering if we could rewrite this block to make what `locale === 'en' ? head(responses).meta : responses[1].meta` is doing clearer.  Maybe something like:

```js
const [enResponse, cyResponse] = responses;
return {
    meta: locale === 'en' ? enResponse.meta : cyResponse.meta,
    result: mergeWelshBy('slug')(
        locale,
        enResponse.map(mapAttrs),
        cyResponse.map(mapAttrs)
    ),
};
```

Reply:
Good call – needed to be `mapAttrs(enResponse)` but much clearer 👍

(FYI the reason for this change was to allow the `meta` field to return the current locale's version, which is now translateable for the global)